tests/tests/test_regressions.py

```diff
@@ -50,32 +76,30 @@ def _reproduce_test(path_to_json: Path, scenario: Scenario, rolling_stock_id: in
     fuzzer_output = json.loads(path_to_json.read_bytes())
     _load_infra(EDITOAST_URL, scenario.infra)
 
+    assert fuzzer_output["infra_name"] in ["small_infra", "Small Infra"]
+    timetable = scenario.timetable
+
     if fuzzer_output["error_type"] == "STDCM":
+        _apply_prelude(fuzzer_output.get("prelude", []), EDITOAST_URL, scenario.infra, timetable, rolling_stock_id)
         _stdcm_with_payload(EDITOAST_URL, fuzzer_output["stdcm_payload"])
```

Comment:
The stdcm payload needs to be modified a little now, it needs to have the right infra / rollingstock / timetable ids

Reply:
Thanks. Apparently it's not enough to reproduce 500 errors from the fuzzer, inconsistent rolling stock might be an issue, I'll investigate a bit more.

Reply:
Indeed it works best when consistently using fast_rolling_stock, let me see what I can do.